### DISCUSSION TITLE
[Q/A][Java SDK (Component)] dubbo Version:2.7.8,load balancing

### DISCUSSION THREAD

**Discussion by khy666** [apache/dubbo#13946] [Question]
### Pre-check

- [x] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

spring-cloud-starter-alibaba-nacos-discovery 2.2.6.RELEASE,
nacos Client Version.:2.0.3 ，
nacos Version 2.7.8，Setting weights with the nacos client does not work
![微信图片_20240319171411](https://github.com/apache/dubbo/assets/49702205/d4080bc2-4045-4853-a5b8-c29bee993793)

dubboDoes not use the weight parameter set by nacos (the client offline function works, but it cannot accept the return result of the offline application when the service still has an ongoing request)
The default is to use RandomLoadBalance to randomize,
 
![微信图片_20240319171402](https://github.com/apache/dubbo/assets/49702205/106e9d86-f01d-42a3-a81b-ca88fb7e9994)
![微信图片_20240319171408](https://github.com/apache/dubbo/assets/49702205/21186af3-337e-4c6f-b709-0ec1bf799d9a)

But this version only configures weights for service methods, how do I configure application level weights and configure them dynamically.
### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please re-submit the discussion in English. Collaboration in Github is only in English
